apps/web/app/(roles)/api/vagas/cron/route.ts

```diff
@@ -0,0 +1,32 @@
+import { getSupabaseClient } from 'db'
+import { Resend } from 'resend'
+
+// eslint-disable-next-line turbo/no-undeclared-env-vars
+const CRON_SECRET = process.env.CRON_SECRET
+// eslint-disable-next-line turbo/no-undeclared-env-vars
```

Comment:
Why is this happening? 

Is there a way to declare the env somewhere and NOT disable the eslint rule?